2019-10-12 13:38:38,723 ERROR: Exception on /index [GET] [in /home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/knerr/repos/mydrop/mytrilog/app/routes.py", line 39, in index
    prev_url=prev_url)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/templating.py", line 140, in render_template
    ctx.app,
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/knerr/repos/mydrop/mytrilog/app/templates/index.html", line 3, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "/home/knerr/repos/mydrop/mytrilog/app/templates/base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/home/knerr/repos/mydrop/mytrilog/app/templates/base.html", line 39, in block "content"
    {% block app_content %}{% endblock %}
  File "/home/knerr/repos/mydrop/mytrilog/app/templates/index.html", line 10, in block "app_content"
    {{ wtf.quick.form(form) }}
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/jinja2/environment.py", line 430, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'jinja2.environment.TemplateModule object' has no attribute 'quick'
2019-10-12 13:39:36,660 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:45]
2019-10-12 13:44:33,650 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:45]
2019-10-15 07:37:39,145 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:45]
2019-10-15 07:55:26,643 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:45]
2019-10-15 07:56:24,042 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:47]
2019-10-15 08:13:42,353 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:47]
2019-10-15 08:41:53,256 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:47]
2019-10-15 08:45:36,601 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:47]
2019-10-15 08:47:34,066 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:47]
2019-10-15 08:48:31,594 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:47]
2019-10-15 08:48:47,642 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:47]
2019-10-15 08:49:28,044 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:47]
2019-10-15 08:49:48,557 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:47]
2019-10-15 09:13:38,054 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:47]
2019-10-15 12:42:13,731 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:49]
2019-10-15 12:47:17,873 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:49]
2019-10-18 09:34:21,487 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:49]
2019-10-18 09:49:52,656 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:49]
2019-10-18 09:49:56,552 ERROR: Exception on /index [GET] [in /home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/knerr/repos/mydrop/mytrilog/app/routes.py", line 41, in index
    prev_url=prev_url)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/templating.py", line 140, in render_template
    ctx.app,
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/knerr/repos/mydrop/mytrilog/app/templates/index.html", line 3, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "/home/knerr/repos/mydrop/mytrilog/app/templates/base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/home/knerr/repos/mydrop/mytrilog/app/templates/base.html", line 46, in block "content"
    {% block app_content %}{% endblock %}
  File "/home/knerr/repos/mydrop/mytrilog/app/templates/index.html", line 16, in block "app_content"
    {% include '_workout-small.html' %}
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/home/knerr/repos/mydrop/mytrilog/venv/lib/python3.5/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: _workout-small.html
2019-10-18 09:50:17,209 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:49]
2019-10-18 09:52:47,004 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:49]
2019-10-18 09:57:07,771 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:49]
2019-10-18 10:00:44,811 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:49]
2019-10-18 10:07:57,698 INFO: MyTriLog startup... [in /home/knerr/repos/mydrop/mytrilog/app/__init__.py:49]
